addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -486,8 +490,7 @@ def _invoice_constraints_peppol_en16931_ubl(self, invoice, vals):
             constraints.update({
                 'no_customer_bronnoysund': _(
                     "The supplier %s must have a Bronnoysund company registry.",
-                    vals['customer'].display_name
-                ) if 'l10n_no_bronnoysund_number' not in vals['customer']._fields or not vals['customer'].l10n_no_bronnoysund_number else "",
+                    vals['vals']['accounting_customer_party_vals']['party_vals']['contact_vals']['name']
+                ) if not vals['vals']['accounting_customer_party_vals']['party_vals']['endpoint_id'] else '',
             })
-
```

Comment:
I don't think this diff is required: for the supplier, we always use the commercial_partner_id.

Reply:
I'm a bit confused here as the error message is referring to the 'supplier' while the condition is based on 'customer' data. If we are actually referring about the customer, then this modification is necessary, else I'll remove it. Since constraints on the supplier are present in lines just above, I assume those are actually for the customer. I'll update the error message.